master/static/migrations/20220311095512_add_workspace_tables.up.sql

```diff
@@ -0,0 +1,53 @@
+-- Tables
+CREATE TABLE workspaces (
+  id SERIAL PRIMARY KEY,
+  name TEXT UNIQUE NOT NULL,
+  archived BOOLEAN,
+  created_at timestamp with time zone NOT NULL DEFAULT NOW(),
+  user_id INT REFERENCES users(id),
+  immutable BOOLEAN DEFAULT FALSE
+);
+CREATE TABLE projects (
+  id SERIAL PRIMARY KEY,
+  name TEXT NOT NULL,
+  description TEXT,
+  archived BOOLEAN,
+  created_at timestamp with time zone NOT NULL DEFAULT NOW(),
+  notes JSONB,
```

Comment:
I think we want this as a JSONB array. That would let us store multiple "pages" of notes as in the initial design.